### PR TITLE
Update intro with minor text changes and some links

### PR DESCRIPTION
**Pull request by mlvandijk**

### CODE REVIEW

**Comment by mlvandijk** [comment]
@plaindocs Some minor changes (didn't brake the build this time). Let me know what you think.

File: content/introduction_to_cucumber_and_bdd.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ Each scenario is a list of steps for Cucumber to work through. So that Cucumber
 can understand these feature files, they must follow some basic syntax rules.
 The name for this set of rules is [Gherkin](/gherkin/gherkin-intro/).
 
-Along with the features, you give Cucumber a set of step definitions. These
-files map each business-readable language step into programming code to carry
+Along with the features, you give Cucumber a set of [step definitions](/cucumber/step-definitions/). These
+files map ("glue") each business-readable language step into programming code to carry
```
Comment (Zearin) [inline content/introduction_to_cucumber_and_bdd.md]: Perhaps change 
> ("glue")

to

> (or "glue")

**Review by Zearin — COMMENTED** [review]
Looks good overall!  (Otherwise, see my line comment on the Intro doc)

File: content/introduction_to_cucumber_and_bdd.md (hunk 1 above)
Comment (mlvandijk) [reply content/introduction_to_cucumber_and_bdd.md]: Yes, that's better. Thanks!

**Comment by mlvandijk** [comment]
@Zearin Thanks for the review. One minor change, as you suggested! :)

**Comment by mlvandijk** [comment]
Deploy was failing due to heavy load on http://pragprog.com (see comment in https://github.com/cucumber/docs.cucumber.io/pull/21)
Now that http://pragprog.com is back up, deploy works fine. \o/

**Comment by mlvandijk** [comment]
Close this PR; will redo on master.

**Pull request closed without merging**